src/kubernetes-ingress-controller/guides/using-gateway-api.md

```diff
@@ -93,6 +93,7 @@ by all Gateways of a given type.
 
 Add a GatewayClass:
 
+{% if_version lte: 2.5.x %}
```

Comment:
Can't we use something like `lt: 2.6.x`? In a possible scenario where we release 2.5.1 this will present the accurate state of things.

Reply:
I honestly don't know if e.g. 2.5.1 would match `lte: 2.5.x`

Reply:
I'll take a try later.

Reply:
`2.5.x` would match `2.5.1`